src/types/Types.ts

```
@@ -31,6 +31,8 @@ interface Social {
 interface Skill {
   title: string;
   level: number;
+  icon: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)